projects.go

```diff
@@ -34,6 +35,14 @@ func (p *ProjectService) ListProjects(params map[string]string) ([]*Project, *Li
 
 // CreateProject creates an awx project.
 func (p *ProjectService) CreateProject(data map[string]interface{}, params map[string]string) (*Project, error) {
+	mandatoryFields = []string{"name", "organization", "scm_type"}
```

Comment:
ONLY `name` is required, see https://docs.ansible.com/ansible-tower/latest/html/towerapi/projects.html#reviewing-the-options-endpoint

Reply:
I followed the doc but received 400. So I checked in gui.
<img width="1415" alt="screen shot 2018-08-28 at 16 45 55" src="https://user-images.githubusercontent.com/13198324/44730657-c2f50080-aae1-11e8-8db4-2f812c144682.png">

Reply:
Okay, let's just merge, and let this as another PR's task!

Reply:
Okay, let's just merge and leave this as another PR's task.